notebook/static/auth/js/loginmain.js

```diff
@@ -5,9 +5,11 @@ define(['jquery', 'base/js/namespace', 'base/js/page'], function($, IPython, pag
     function login_main() {
       var page_instance = new page.Page('div#header', 'div#site');
       $('button#login_submit').addClass("btn btn-default");
+      $('button#login_new_pass_submit').addClass("btn btn-default");
```

Comment:
I think these classes could just be written directly in the HTML, rather than being added by Javascript. I can see it's copied from the line above, but I don't know why that's done in Javascript either.

Reply:
sure thing, we’ll fix it!

Reply:
@danagilliann and I will be working on this!